Sources/BraveWallet/WalletStrings.swift

```diff
@@ -3053,6 +3053,13 @@ extension Strings {
       value: "Approve Transaction",
       comment: "The title displayed above the value of a Solana dapp transaction in transaction confirmation view, transaction details view and transaction summary rows."
     )
+    public static let solanaSwapTransactionTitle = NSLocalizedString(
+      "wallet.solanaDappTransactionTitle",
```

Comment:
```suggestion
      "wallet.solanaDappTransactionTitle","wallet.solanaSwapTransactionTitle"
```

Reply:
Fixed in [9ee57fc](https://github.com/brave/brave-ios/pull/6915/commits/9ee57fc395ce22ca2409ed5dd82509857868afe2), nice catch